### PR TITLE
refactor: Issuer API BDD tests

### PR DESCRIPTION
**Pull request by rolsonquadras**
- Added scenario for issueCred to be inline with composeAndIssueCred

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: test/bdd/features/issuer_e2e_api.feature [hunk 1]
```diff
@@ -10,11 +10,16 @@ Feature: Issuer VC REST API
...
-    Then  Verify the proof value generated using the Issuer Service Issue Credential API with the DID stored in "did" variable
+    Given "University" has a DID
+    Then  "University" application service verifies the credential created by Issuer Service issueCredential API with it's DID
 
   @composeAndIssueCred_api
   Scenario: Compose and Issue Credential API
-    Given "University" has a did
-    Then  "University" application service verifies the credential created by Issuer Service Issue Credential API
+    Given "University" has a DID
+    Then  "University" application service verifies the credential created by Issuer Service composeAndIssueCredential API with it's DID
```
Comment (sandrask) [inline test/bdd/features/issuer_e2e_api.feature]: we should not be using composeAndIssueCredential in BDD test - it should be human readable. 

Given "University" has a DID
Then  "University" application service verifies that the credential created by Issuer Service is created with it's DID

File: test/bdd/pkg/issuer/issuer_steps.go [hunk 2]
```diff
@@ -254,6 +259,12 @@ func (e *Steps) signCredential(didDocVar string) ([]byte, error) {
 	return responseBytes, nil
 }
 
+func (e *Steps) issueCred(user string) error {
```
Comment (sandrask) [inline test/bdd/pkg/issuer/issuer_steps.go]: It is funny that issueCred calls verifyCredential only

File: test/bdd/features/issuer_e2e_api.feature (hunk 1 above)
Comment (rolsonquadras) [reply test/bdd/features/issuer_e2e_api.feature]: I'll update in next PR.

File: test/bdd/pkg/issuer/issuer_steps.go (hunk 2 above)
Comment (rolsonquadras) [reply test/bdd/pkg/issuer/issuer_steps.go]: I'll update in next PR.

**Pull request merged**